test/studies/lcals/LCALSMain.chpl

```diff
@@ -0,0 +1,1203 @@
+module main {
+  use FileSystem;
+  use LCALSParams;
+  use LCALSEnums;
+  use LCALSDataTypes;
+  use LCALSLoops;
+  use LCALSStatic;
+  use LCALSConfiguration;
+
+  use RunARawLoops;
+  use RunBRawLoops;
+  use RunCRawLoops;
+  use RunParallelRawLoops;
+  use RunVectorizeOnlyRawLoops;
+
+  proc allocateLoopData() {
+    const num_aligned_segments =
+      (s_loop_data.max_loop_length + 20)/LCALS_DATA_ALIGN + 1;
+    const aligned_chunksize = num_aligned_segments*LCALS_DATA_ALIGN;
+
+    writeln("\n allocateLoopData...");
```

Comment:
It is in the reference version, and I made an effort to match the reference output as exactly as possible.  There are a few other "doingXYZ..." writelns that fall into this category also.